### ISSUE TITLE
Add a tutorial on using autocompletion with LHCb software

### ISSUE THREAD

**Issue by ibab** [lhcb/second-analysis-steps#41]
@bartoszmalecki was interested in enabling autocompletion with the LHCb software.
We could make a lesson in the second-analysis-steps that explains it.

**Comment by pseyfert** [comment]
you mean things like [pseyfert/lb-bash](https://github.com/pseyfert/lb-bash/) and [pseyfert/lb-zsh](https://github.com/pseyfert/lb-zsh) or something less hacked?

**Comment by ghost** [comment]
Yes, that`s exactly what I`ve been looking for. Thank you so much.

Best regards,
Bartosz


W dniu 20.12.2016 o 13:18, pseyfert pisze:
